Formula/vapor.rb

```diff
@@ -0,0 +1,21 @@
+class Vapor < Formula
+  desc "Command-line tool for Vapor (Server-side Swift web framework)"
+  homepage "https://vapor.codes"
+  url "https://github.com/vapor/toolbox/archive/18.0.0.tar.gz"
+  sha256 "1d1b72d92dfb4e417dbbfe8486149f64a92cbfcf86db5209686fdd926b43c74c"
+  head "https://github.com/vapor/toolbox.git"
+
+  depends_on :xcode => "11.4"
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "-Xswiftc", \
+      "-cross-module-optimization", "--enable-test-discovery"
+    mv ".build/release/vapor", "vapor"
+    bin.install "vapor"
+  end
+
+  test do
+    system "vapor", "new", "hello-world", "-n"
+    assert_predicate "hello-world/Package.swift", :exist?
```

Comment:
`assert_predicate testpath"/hello-world/Package.swift", :exist?` might work better.

And you might also want to use testpath for the project creation.